lib/client-assets.php

```diff
@@ -261,6 +261,13 @@ function gutenberg_register_packages_scripts( $scripts ) {
 				break;
 		}
 
+		// Load unminified JS in SCRIPT_DEBUG mode.
+		if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) {
+			// We can't replace the glob call above with index.js because the PHP
+			// asset file is associated with the .min file. (index.min.asset.php).
+			$path = str_replace( '.min.js', '.js', $path );
```

Comment:
When running `npm run dev` we only generate `.min.js` files, though in reality it's a non minified file. Should we update the `npm run dev` build step to also include `.js` files? maybe we can just avoid breakage in this case and always load the non minified versions.